package.json

```diff
@@ -33,7 +33,7 @@
     }
   },
   "scripts": {
-    "preinstall": "git submodule update --init --recursive && yarn --cwd ccd-definition-processor install",
+    "preinstall": "git submodule update --init --recursive && git submodule foreach --recursive git pull && yarn --cwd ccd-definition-processor install",
```

Comment:
We need to actually pull the changes from git for the sub-module if remote master is ahead of our submodule's commit